crates/log-server/src/service.rs

```diff
@@ -150,87 +150,134 @@ impl LogServerService {
             .await
             .context("Cannot load log-server marker from log-store")?;
         // If we have a marker, it must match our own node id.
-        if let Some(marker) = maybe_marker {
+        if let Some(marker) = &maybe_marker {
             if *marker.id() != my_node_id.as_plain() {
                 return Err(anyhow::anyhow!(
-                    "LogStoreMarker doesn't match our own node-id. Found NodeId {} while our node is {}",
+                    "log-server data marker was found but it was created by another node. Found {} while this node is {}",
                     marker.id(),
                     my_node_id.as_plain()
                 ));
             } else {
                 debug!(
-                    "Found matching LogStoreMarker in log-store, written at '{:?}'",
-                    marker.created_at()
+                    "Found log-server data marker, written at '{}'",
+                    marker.created_at().to_rfc2822()
```

Comment:
Could you update https://github.com/restatedev/restate/blob/b8081d451ba5e6dd1fdb3b48a3dc27cbd509d30a/crates/metadata-server/src/raft/server.rs#L190 as well?

Reply:
Done. Let me know if you don't like the changes there.